src/actions/UpdateApiActions.js

```diff
@@ -109,6 +109,24 @@ export default function UpdateApiActions(
         }
     });
 
+    this.postByHref = (href, body = '') => ({
+        [RSAA]: {
+            endpoint: `${appRoot}${href}`,
+            method: 'POST',
+            options: { requiresAuth: true },
+            headers: {
+                Accept: 'application/json',
```

Comment:
you could make this support more than just json bodies if you feel like it. Feel free to merge without though